crates/orchestrator/src/queue/sqs/mod.rs

```diff
@@ -1,15 +1,30 @@
+use std::collections::HashMap;
+use std::string::ToString;
 use std::time::Duration;
 
-use crate::queue::job_queue::JOB_PROCESSING_QUEUE;
+use crate::queue::job_queue::{
+    JOB_HANDLE_FAILURE_QUEUE, JOB_PROCESSING_QUEUE, JOB_VERIFICATION_QUEUE, WORKER_TRIGGER_QUEUE,
+};
 use async_trait::async_trait;
 use color_eyre::Result;
+use lazy_static::lazy_static;
 use omniqueue::backends::{SqsBackend, SqsConfig, SqsConsumer, SqsProducer};
 use omniqueue::{Delivery, QueueError};
 use utils::env_utils::get_env_var_or_panic;
 
 use crate::queue::QueueProvider;
 pub struct SqsQueue;
 
+lazy_static! {
+    /// Maps Queue Name to Env var of queue URL.
+    pub static ref QUEUE_NAME_TO_ENV_VAR_MAPPING: HashMap<String, String> = HashMap::from([
+        (JOB_PROCESSING_QUEUE.to_string(), "SQS_JOB_PROCESSING_QUEUE_URL".to_string()),
+        (JOB_VERIFICATION_QUEUE.to_string(), "SQS_JOB_VERIFICATION_QUEUE_URL".to_string()),
+        (JOB_HANDLE_FAILURE_QUEUE.to_string(), "SQS_JOB_HANDLE_FAILURE_QUEUE_URL".to_string()),
+        (WORKER_TRIGGER_QUEUE.to_string(), "SQS_WORKER_TRIGGER_QUEUE_URL".to_string()),
```

Comment:
I don't see any reason to convert these to strings, seems like you could be happy with a `HashMap<&str, &str>`?

Reply:
done